cpp/test/Slice/errorDetection/WarningInvalidComment.err

```diff
@@ -4,16 +4,6 @@ WarningInvalidComment.ice:11: warning: the '@exception' tag is only valid on ope
 WarningInvalidComment.ice:11: warning: the '@return' tag is only valid on operations
 WarningInvalidComment.ice:15: warning: missing link target after '@see' tag
 WarningInvalidComment.ice:17: warning: ignoring trailing '.' character in '@see' tag
-WarningInvalidComment.ice:43: warning: '@param NoParam' does not correspond to any parameter in operation 'voidOp'
```

Comment:
Why are we removing these warnings?

Reply:
Meant to comment this in the file... but these warnings were duplicates.
If you look at the bottom of this file, you'll see they appear a 2nd time.

This was because slice2cpp parsed some comments twice (in different visitors).

With the new setup we guarantee doc-comments are parsed exactly once.
Eliminating this duplication.